qa/tasks/cephfs/test_fragment.py

```diff
@@ -314,3 +314,78 @@ def _count_fragmented():
             lambda: _count_fragmented() > 0,
             timeout=30
         )
+
+class TestDirfragKillPoints(TestFragmentation, CephFSTestCase):
+    # Two active MDS
+    MDSS_REQUIRED = 2
+
+    init = False
+
+    def _setup_cluster(self):
+        # Set Multi-MDS cluster
+        self.fs.set_max_mds(2)
+
+        self.fs.wait_for_daemons()
+
+        all_daemons = self.fs.get_daemon_names()
+
+        return True
+
+    def _run_dir_frag(self, killpointv):
+
+        status = self.fs.status()
+
+        rank_0 = self.fs.get_rank(status=status, rank = 0)
+        rank_1 = self.fs.get_rank(status=status, rank = 1)
+        rank_0_name = rank_0['name']
+        rank_1_name = rank_1['name']
+
+        command = ["config", "set", "mds_kill_dirfrag_at", str(killpointv)]
+        result = self.fs.rank_asok(command, rank=0, status=status)
+        assert(result["success"])
+
+        try:
+            self.test_oversize()
+        except Exception as e:
+            log.error(e.__str__())
+
+        status2 = self.fs.status()
+
+        rank_0_new = self.fs.get_rank(status=status2, rank = 0)
+        rank_1_new = self.fs.get_rank(status=status2, rank = 1)
+        rank_0_new_name = rank_0_new['name']
+        rank_1_new_name = rank_1_new['name']
+
+        if status2.hadfailover_rank(self.fs.id, status, 0):
+            log.info("MDS %s crashed and active as MDS %s at killpoint %d"
+                    %(rank_0_name, rank_0_new_name, killpointv))
+
+        if status2.hadfailover_rank(self.fs.id, status, 1):
+            log.info("MDS %s crashed and active as MDS %s killpoint %d"
+                    %(rank_1_name, rank_1_new_name, killpointv))
```

Comment:
```suggestion
                        self.fail(f"MDS {rank_1_name} crashed and active as MDS {rank_1_new_name} killpoint {killpointv}"
```

Reply:
We don't want to fail the test case here right? since the mds crash and recovery was supposed to happen